Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:14-alpine AS build
+
+WORKDIR /usr/src/app
+
+COPY package.json .
+COPY tsconfig.json ./
+RUN npm install
+
```

Comment:
i think it would be simpler to run `npm run build` and copy the `/dist` container from `build` stage into the next stage.
What happens now you are copying the first container into the 2nd one which not ideal for the multistage build.

In the 2nd stage you could set up a python container that servers the `dist` folder no need for `npm` e.g. how it is done in e2e tests: `python -m http.server 8081 --d ./dist`